tests/main/snap-user-services-mixed-refresh-single-disabled/task.yaml

```diff
@@ -90,22 +90,28 @@ execute: |
 
     # stop and disable user services, but the scope of the disable
     # for users should only affect the current user.
+    # OBS: Currently this will not work, the way user-services are enabled
+    # is by --global enable, which means that any attempt to individually disable
+    # services will be a no-op.
+    # TODO: Switch to a user-preset approach and scrap the --global enable/disable
+    # for user-services specifically
```

Comment:
we probably can't scrap the global enable/disable but need to have a way to override per user, no? or just error if we get --user and --enable/--disable ?

Reply:
I will update this comment - this is no longer really valid as we want to maybe support user masking of services instead of preset. I don't think preset is the right way to go anymore